src/prop/model.py

```diff
@@ -127,24 +127,28 @@ def LandClassification(self, lat, lng):
 
   # This is the oracle for propagation loss from point 1 to point 2 at frequency f (Mhz).
   def PropagationLoss(self, f, lat1, lng1, h1, lat2, lng2, h2, land_cat=''):
+  # Note: lat1/lng1/h1 _must_ correspond to the CBSD for proper implementation of R2-SGN-04
+
     if land_cat == '':
       code = self.nlcdIndx.NlcdCode(lat2, lng2)
       if code == 11:
         code = self.nlcdIndx.NlcdCode(lat1, lng1)
       land_cat = land_use.NlcdLandCategory(code)
     print 'Using land_cat =', land_cat
 
-#   Calculate effective heights of tx and rx:
+    # Note: R2-SGN-04 calls for a minimum CBSD height of 20 m for all eHata calculations.
+
+    # Calculate effective heights of CBSD and rx:
     profile = self.nedIndx.Profile(lat1, lng1, lat2, lng2)
-    h1eff, h2eff = EffectiveHeights(h1, h2, profile)
+    h1eff, h2eff = EffectiveHeights(max(h1,20.), h2, profile)
```

Comment:
It's not clear. I honestly don't think SGN-04 meant to say that h_b should be h_effective. Maybe they did, but look at it this way -- it then says that if you have two devices separated by 50 m, then the CBSD side used effective height and the other side uses terrain site, in the slant-range calculation. That makes no sense. There's no way that "effective height" on one end could have any influence on FSL between two devices on the opposite side of a street from each other. That's silly.